metals/src/main/scala/scala/meta/internal/rename/RenameProvider.scala

```diff
@@ -144,12 +146,14 @@ final class RenameProvider(
         Seq(uri.toAbsolutePath -> textEdits.toList)
       }
       val fileChanges = allChanges.flatten.toMap
+      val openFilesOnRenameProvider =
+        clientExperimentalCapabilities.openFilesOnRenameProvider
```

Comment:
We could even do:
```
val openFilesOnRenames = metalsConfig.openFilesOnRenames || clientExperimentalCapabilities.openFilesOnRenameProvider
```

Reply:
Sorry for not being specific :(

Reply:
Ah, duh. Of course. No worries, that was my bad, just quickly doing it and not thinking.